src/main/java/sirius/biz/storage/layer2/jdbc/SQLProcessBlobChangesLoop.java

```diff
@@ -31,7 +31,7 @@ public class SQLProcessBlobChangesLoop extends ProcessBlobChangesLoop {
 
     @Override
     protected void deleteBlobs(Runnable counter) {
-        buildBaseQuery(SQLBlob.class, query -> query.eq(SQLBlob.DELETED, true)).iterateAll(blob -> {
+        buildBaseQuery(SQLBlob.class, query -> query.eq(SQLBlob.DELETED, true)).streamBlockwise().forEach(blob -> {
```

Comment:
Do you need this here? buildBaseQuery set a limit of 1024

Reply:
Generally it is a bad idea to keep the connection open while waiting for file operations. This change will free up resources earlier.

Reply:
unfortunately its using SmartQuery here and at some other places, where streamBlockwise is not working, so i´ll revert it.

Reply:
what about reducing the limit to 1000 and use queryList()?